Automattic/wp-calypso
# Editor Deprecation: Consolidate the editor URL selectors

#### Changes proposed in this Pull Request

Now that the only editor is Gutenberg, we are more concerned with
whether we should load the editor in the Gutenframe or redirect them to
`wp-admin`.

There were two selectors to get the appropriate editor URL,
`getGutenbergEditorUrl` and `getEditorUrl`. They were essentially the
same with subtle differences. This change consolidates them as
`getEditorUrl` and updates any references to `getGutenbergEditorUrl`.

#### Testing instructions

* Check that links to edit posts and pages don't change for simple sites, and sites that are eligible for Gutenframe, with this change.
* For Atomic sites that are ineligible for Gutenframe, check that the `calypsoify=1` parameter is added to the editor URLs.

#### Calypsoify Parameter

As we use the Calypsoify parameter to indicate if someone has come from Calypso. Should we set it on all editor URLs?
